src/fonts/OTF/TeX/Makefile

```diff
@@ -75,9 +72,9 @@ pfa: $(MFTRACE_MODIFIED)
 	$(PYTHON) $(MFTRACE_MODIFIED) --magnification 1000 --simplify cmssbx10
 
 	@echo "eufm10"
-	$(PYTHON) $(MFTRACE_MODIFIED) --magnification 1000 --simplify eufm10
+	cp "`$(KPSEWHICH) eufm10.pfb`" eufm10.pfb
 	@echo "eufb10"
-	$(PYTHON) $(MFTRACE_MODIFIED) --magnification 1000 --simplify eufb10
+	cp "`$(KPSEWHICH) eufb10.pfb`" eufb10.pfb
```

Comment:
Why don't we need to trace these fonts anymore?

Reply:
@kevinbarabash Euler fonts were [reshaped as of version 3.0](http://www.tug.org/TUGboat/tb29-2/tb92hagen-euler.pdf) and since the updates were made directly to the Type 1 files, MetaFont sources have been removed. So there are no MetaFont sources to trace.